libraries/AP_HAL_Linux/UARTDriver.cpp

```diff
@@ -284,44 +247,33 @@ void UARTDriver::set_blocking_writes(bool blocking)
  */
 bool UARTDriver::tx_pending()
 {
-    return !BUF_EMPTY(_writebuf);
+    return _writebuf.available();
```

Comment:
I know it's not needed, but it would be better if this returned an actual bool via .available>0.